apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -383,11 +383,9 @@ export class Utilities {
   ): void {
     if (fsx.existsSync(directory)) {
       console.log('Deleting old files from ' + directory);
-      Utilities.dangerouslyDeletePath(directory);
+      fsx.emptyDirSync(directory);
```

Comment:
>emptyDirSync [](start = 10, length = 12)

What's the performance on this vs the other method?

Reply:
Well, it would be faster in the case where createFolderWithRetry() spins for a few seconds waiting for the virus scanner.

But the reason I changed it was because the tool was actually crashing with an error.  Windows really doesn't like it when you delete+recreate a folder.  fsx.emptyDirSync() empties the folder contents and creates the folder if it doesn't exist, which is generally a much better way to go.

---
In reply to: [185169033](https://github.com/Microsoft/web-build-tools/pull/632#discussion_r185169033) [](ancestors = 185169033)